problemsets-mat223/modules/module1-exercises.tex

```diff
@@ -126,6 +126,52 @@
     		    \item Yes.
 		    \end{enumerate}
 		\end{solution}
+		
+		\prob
+		    Use set notations, vectors, or linear combinations of vectors to describe the following sets.
+		    \begin{enumerate}
+		        \item The set of vectors with rational components in $\R^2$.
+		        \item The set of vectors with irrational components in $\R^2$.
+		        \item The set that contains the empty set.
+		        \item $\R^3$.
+		    \end{enumerate}
+    		
 		% Challenge (3 questions)
+		\prob 
+		% Propose: get students to understand quantifiers and set operations
+		% practice providing justification to their intuitive guesses
+		Which of the following statements is true about the set listed below? Justify your answers.
+		    \begin{enumerate}
+		        \item The y-axis in $\R^3$.
+		        \begin{enumerate}
+		            \item It can be written as a set of linear combinations of vectors.
+		            \item There exists sets $A , B \subseteq \R^3$ such that this set is equal to $A \cup B$.
+		            \item There exists sets $A , B \subseteq \R^3$ such that this set is equal to $A \cap B$.
+		            \item For any vectors $\vec v \in \emptyset$, $\vec v \neq \vec 0$
```

Comment:
What does this have to do with the set? It should be a separate question. Use `For all` instead of `For any`.

Reply:
Make Line 148-149 to a separate question: use set operations to describe some sets?

Reply:
Oh sorry this are typos. It should be vectors in the set of $\mathcal Y$ which is the y-axis.